md/api.py

```diff
@@ -0,0 +1,138 @@
+from django.db import connections
+from django.db.models import Count, Q
+
+from rest_framework import viewsets
+from rest_framework.decorators import detail_route
+from rest_framework.response import Response
+from rest_framework_extensions.cache.decorators import cache_response
+from rest_framework_extensions.key_constructor import bits
+from rest_framework_extensions.key_constructor.constructors import DefaultObjectKeyConstructor
+
+from md.models import Agency, Stop
+from md import serializers
+from md.utils import GroupedData
+
+
+GROUPS = {'A': 'ASIAN',
+          'B': 'BLACK',
+          'I': 'NATIVE AMERIOCAN',
+          'U': 'OTHER',
+          'W': 'WHITE',
+          'H': 'HISPANIC'}
+
+# PURPOSE_CHOICES = {1: 'Speed Limit Violation',
```

Comment:
Regarding all the commented-out code taken from NC:

If it were me, I'd leave hints in comments for anything I learned about the code not currently used, but I wouldn't leave it here commented-out.  The main reason is that the person making it work might not think to go find the latest version of the NC code.